### ISSUE TITLE
[feature] DOT language support

### ISSUE THREAD

**Issue by ayalpani** [tldraw/tldraw#1079]
Hi Steve, thanks for tldraw. Did you ever think about supporting DOT Language for automatically create Graphs?

`$ echo 'digraph { a -> b }' | tldraw > mygraph.tldr`

**Comment by linonetwo** [comment]
I think this can be done by writing a compiler that generates tldr json format. You can try build one yourself as 3rd party project using https://github.com/antlr/antlr4

**Comment by steveruizok** [comment]
Yes, we've done some prototyping on a feature like this. It worked quite well! But there will be a lot more work to do to go from prototype to feature. I'd like to get a start on it, then release and see where the community takes it.
